supabase/sql/advisor_fk_covering_indexes.sql

```
@@ -0,0 +1,31 @@
+-- ITX-50: add covering indexes for unindexed foreign keys
+-- (advisor 0001_unindexed_foreign_keys). Speeds up joins and ON DELETE/UPDATE
+-- cascade checks. Exactly the 26 FKs the advisor flagged (FKs already covered
+-- by a leading-column composite index are excluded).
+
+create index if not exists idx_app_runtime_config_updated_by on public.app_runtime_config (updated_by);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)